app/registries/foreman/plugin/importer_registry.rb

```diff
@@ -0,0 +1,26 @@
+module Foreman
+  class Plugin
+    class ImporterRegistry
+      attr_accessor :importers 
+
+      def initialize
+        @importers = {}
+        @importers.default = 'PuppetFactImporter'
+      end
+
+      def register(key, klass, default = false)
```

Comment:
What about adding an option to register a class as a constant, not only as string:

```ruby
importers.register(:puppet, PuppetFactImporter, true)
```
It's more readable, less prone for typos and more intuitive.
To support this, you don't have to store constants in the registry:

``` ruby
def register(key, klass, default = false)
  klass = klass.to_s
  #...
end
```